docs/Actions.md

```diff
@@ -1205,6 +1205,22 @@ Quickly get the name of the branch you're currently in
 git_branch
 ```
 
+
+### git_add
+
+To simply commit one file with a certain commit message use
+
+```ruby
+git_add(path: "./version.txt")
+```
+
+To commit several files with a certain commit message use
+
+```ruby
+git_commit(path: ["./version.txt", "./changelog.txt"])
```

Comment:
should this be `git_add` ? and the comments look like copy pasta too, would you mind changing them to be `add` instead of `commit` and get rid of the commit message stuff?